From 3ef26b708725b528a1c69ab3eb523036c50b89d6 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Tue, 24 Sep 2024 13:05:48 +0200
Subject: [PATCH] misc: Link tst-mkstemp-fuse-parallel with
 $(shared-thread-library)
Content-type: text/plain; charset=UTF-8

The barrier functions require this on Hurd.
---
 misc/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc/Makefile b/misc/Makefile
index 1422c95317..a932b1aab4 100644
--- a/misc/Makefile
+++ b/misc/Makefile
@@ -178,3 +178,4 @@ $(objpfx)tst-select: $(librt)
 $(objpfx)tst-select-time64: $(librt)
 $(objpfx)tst-pselect: $(librt)
 $(objpfx)tst-pselect-time64: $(librt)
+$(objpfx)tst-mkstemp-fuse-parallel: $(shared-thread-library)
-- 
2.43.5

